rubyric/index.rst

```diff
@@ -4,8 +4,7 @@ Module 6 - Rubyric
 Documentation of Rubyric feedback service.
 
 .. toctree::
-  01_introduction
-  02_getting_started
-  03_rubrics_and_reviews
-  04_roles_and_groups
-  example
+  introduction
+  getting_started
+  rubrics_and_reviews
+  roles_and_groups
```

Comment:
Was the `example` chapter removed by accident?

Reply:
Sorry about that. Now should be ready.